components/translation/usage.rst

```diff
@@ -411,17 +411,19 @@ use for translation::
             'fr_FR'
         );
 
-Retrieving the Message Catalogue
---------------------------------
+.. _retrieving-the-message-catalogue:
 
-In case you want to use the same translation catalogue outside your application
+Retrieving the Message Catalog
+------------------------------
+
+In case you want to use the same translation catalog outside your application
```

Comment:
outside of ?

Reply:
https://english.stackexchange.com/questions/9700/outside-or-outside-of

So this is a real hard one. Seems like "outside of" is more American English, but it also has the meaning "except for" and not "outside". So I would say "outside" (without "of") is the right word here.